pkg/conversion/meta.go

```diff
@@ -59,7 +59,13 @@ func (SimpleMetaFactory) Interpret(data []byte) (version, kind string, err error
 	}{}
 	err = json.Unmarshal(data, &findKind)
 	if err != nil {
-		return "", "", fmt.Errorf("couldn't get version/kind: %v", err)
+		return "", "", fmt.Errorf("unable to parse json: %v", err)
```

Comment:
This is the right place, but please don't add the below two checks (notice the tests fail now-- those aren't actually error conditions).

I'd recommend making this error message read thusly:
"couldn't get version/kind; json parse error: %v"